gm4_end_fishing/data/gm4_end_fishing/functions/reel_in/lore/sword.mcfunction

```diff
@@ -1,23 +1,23 @@
+execute if score @s gm4_ef_data matches 10..14 run data merge entity @s {Item:{tag:{gm4_ef_sword:3b,display:{Lore:['"Blessed with Hylia\'s power"']}}}}
+execute if score @s gm4_ef_data matches 15..19 run data merge entity @s {Item:{tag:{gm4_ef_sword:4b,display:{Lore:['"Forged by Muramasa"']}}}}
+execute if score @s gm4_ef_data matches 20..24 run data merge entity @s {Item:{tag:{gm4_ef_sword:5b,display:{Lore:['"Legend says this could be found"','"inside the remains of Outpost 17"']}}}}
+execute if score @s gm4_ef_data matches 25..29 run data merge entity @s {Item:{tag:{gm4_ef_sword:6b,display:{Lore:['"Used to decapitate the Gorgon, Medusa"']}}}}
+execute if score @s gm4_ef_data matches 30..34 run data merge entity @s {Item:{tag:{gm4_ef_sword:7b,display:{Lore:['"This is believed to be one of the"','"finest Japanese swords ever made"']}}}}
+execute if score @s gm4_ef_data matches 35..39 run data merge entity @s {Item:{tag:{gm4_ef_sword:8b,display:{Lore:['"Used at the Battle of the Trench"']}}}}
+execute if score @s gm4_ef_data matches 40..44 run data merge entity @s {Item:{tag:{gm4_ef_sword:9b,display:{Lore:['"Removed from the Garden of Eden"']}}}}
+execute if score @s gm4_ef_data matches 45..49 run data merge entity @s {Item:{tag:{gm4_ef_sword:10b,display:{Lore:['"Used to kill the dragon Fafnir"']}}}}
+execute if score @s gm4_ef_data matches 50..54 run data merge entity @s {Item:{tag:{gm4_ef_sword:11b,display:{Lore:['"Snicker-snack!"']}}}}
+execute if score @s gm4_ef_data matches 55..59 run data merge entity @s {Item:{tag:{gm4_ef_sword:12b,display:{Lore:['"Created by husband and"','"wife magicians of the"','"ancient Bön tradition"']}}}}
+execute if score @s gm4_ef_data matches 60..64 run data merge entity @s {Item:{tag:{gm4_ef_sword:13b,display:{Lore:['"Belongs to Roland"']}}}}
+execute if score @s gm4_ef_data matches 65..69 run data merge entity @s {Item:{tag:{gm4_ef_sword:14b,display:{Lore:['"If found, please return"','"to William Wallace"']}}}}
+execute if score @s gm4_ef_data matches 70..74 run data merge entity @s {Item:{tag:{gm4_ef_sword:15b,display:{Lore:['"Reforged from the shards of Narsil"']}}}}
+execute if score @s gm4_ef_data matches 75..79 run data merge entity @s {Item:{tag:{gm4_ef_sword:16b,display:{Lore:['"Forged by the Iron Sisters"']}}}}
+execute if score @s gm4_ef_data matches 80..84 run data merge entity @s {Item:{tag:{gm4_ef_sword:17b,display:{Lore:['"Used to lead the revolutionary"','"army against the Telmarines"']}}}}
+execute if score @s gm4_ef_data matches 85..89 run data merge entity @s {Item:{tag:{gm4_ef_sword:18b,display:{Lore:['"Can only be wielded by a true Gryffindor"']}}}}
+execute if score @s gm4_ef_data matches 90..94 run data merge entity @s {Item:{tag:{gm4_ef_sword:19b,display:{Lore:['"Sought by Thrax"']}}}}
+execute if score @s gm4_ef_data matches 95..99 run data merge entity @s {Item:{tag:{gm4_ef_sword:20b,display:{Lore:['"Wielded by Lion-O"']}}}}
```

Comment:
This works, however, the standard for custom NBT in Gamemode 4 is to have a compound called `gm4_<module id>` within `tag` that contains all module specific NBT.
So a better solution would be to have `{Item:{tag:{gm4_end_fishing:{sword:1b}}}}`

To make these changes work, the loot table would have to be modified as well.

Reply:
Alright, I was following suit with the Audrey messages, do those need changing as well?

Reply:
Oh they use that format as well? Yeah, ideally, but not really high priority - we can make those changes with 1.16, I don't wanna trigger the "Your module is outdated" message for a small backend change

Reply:
We could fix the lack of translations here too?

Reply:
> We could fix the lack of translations here too?

Will do, thanks for reminding me

Reply:
The "module is outdated" message is the least of our concerns. Existing items would no longer work, so in this case I would avoid changing them.

Reply:
In this particular case, I think that's okay. The advancement has the intent of "catch all the swords", not just "touch all the swords". 

We should be more diligent about formats in the future. 
Maybe GM4 needs some update path stuff too. Don't know how that would work.

Reply:
I believe misode is referring to the audrey bottles, as the End Fishing Swords advancement is new to 1.15, thus will have no change.

Reply:
Datafixers for items are hard to do with data packs. We should avoid it at all cost.